src/components/encryption.ts

```diff
@@ -18,10 +20,12 @@ export interface ClientEncryptionSession {
     userId: string;
     deviceId: string;
     accessToken: string;
+    syncToken: string|null;
```

Comment:
We now store the sync token to avoid doing an initial sync on startup, which was causing unbounded amounts of memory to be used.